src/components/custom-select.scss

```diff
@@ -42,7 +55,7 @@
       height: 20px;
       width: 20px;
       margin: 0 5px 0 5px;
-      fill: $cc-teal;
+      fill: var(--theme-primary-color);
```

Comment:
Be cautious when using the theme colors against static backgrounds.  This won't show up with the Interactions theme.  I'm assuming at some point we will get actual theme color specs so we don't need to stress about this now, but something that we should remember to return to when we work on theme colors again.

Reply:
Would it be better if we made them a static color for now then? I debated about it which is why the account owner icon is a static color.

Reply:
As long as it works with the default colors, we're probably safe for now.  I'm assuming we can tackle all of the theme coloring once it is added to the authoring.